persistence-mysql/src/main/resources/migrations/create_tables.sql

```diff
@@ -2,48 +2,6 @@
 
```

Comment:
Lets follow some conventions with our migrations
1. Each migration file should have a single change set
2. Each file must be preceeded by a 00<changeset_number>_<changeset_description>
3. Always have runOnChange:false